app/views/shared/authentication/_providers_registration_form.html.erb

```diff
@@ -1,5 +1,6 @@
 <div class="registration__actions-providers">
   <% authentication_enabled_providers.each do |provider| %>
+  <% binding.break %>
```

Comment:
Still debugging an issue with a previously enabled "twitter" provider stored in `Settings::Authentication.providers`